### ISSUE TITLE
Regarding the validation set during the training process

### ISSUE THREAD

**Issue by fh9892** [cschenxiang/DRSformer#29]
Hello, author. I noticed that the dataset you provided only includes a training set and a test set. Could you please explain how you divided the validation set during the training process?
